sqitch/deploy/instances.sql

```diff
@@ -10,7 +10,8 @@ CREATE TABLE cloudbrain.instances (
 	state         TEXT  NOT NULL,
 	ip_address    TEXT,
 	ssh_key       TEXT,
-	upstream_id   TEXT
+	upstream_id   TEXT,
+	error_reason  TEXT
```

Comment:
tbh I wasn't quite sure how to use sqitch. how to create migrations, how to run them, etc.